students/ghassan/lesson03/mailroom.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import sys
+
+donors = [
+    ('Jessie', (100, 300, 500)),
+    ('James', (343, 22.11)),
+    ('Ford', (3000, 400))]
+
+
+def prompt():
+    choice = input(
+        'Pick one:\n1) Send a thank you\n2) Create a Report\n3) Quit\n')
+    while choice not in ['1', '2', '3']:
+        choice = input(
+            'Pick one:\n1) Send a thank you\n2) Create a Report\n 3) Quit\n')
```

Comment:
it is a good idea to avoid duplicating code, one option is to initialize `choice`, so it would look like:
```
choice = None
while choice not in ['1', '2', '3']:
    choice = input("...")
return choice
```

Also, it is a good idea to choose function names that hint at what it is doing.. for example, `prompt` would instantly tell me an action is being done, but there is not a hint that it will return something 🙂 

something like `get_user_selection` would indicate that we are getting what user selected (`get_` part is the hint)